templates/docs/component-status.md

```diff
@@ -23,18 +23,18 @@ When we add, make significant updates, or deprecate a component we update their
   </thead>
   <tbody>
     <!-- 2.19 -->
+    <tr>
+      <th><a href="/docs/patterns/buttons#small">Small buttons</a></th>
+      <td><div class="p-label--new">New</div></td>
+      <td>2.20.0</td>
+      <td>We added an <code>is-small</code> modifier class for buttons, which can be combined with <code>is-dense</code>.</td>
+    </tr>
     <tr>
       <th><a href="/docs/base/tables#expanding">Expanding table column placeholder</a></th>
       <td><div class="p-label--updated">Updated</div></td>
       <td>2.20.0</td>
```

Comment:
This is a typo actually. It was released in 2.19.2 (.3?) I think. So this as well should drop down to table below.
(and update the version to the correct one).

```suggestion
      <td>2.19.2</td>
```

Reply:
Ah OK - I thought it had crept in there, updated!